registry/recipes/mirror.md

```diff
@@ -30,12 +32,20 @@ relying entirely on your local registry is the simplest scenario.
 
 ### Gotcha
```

Comment:
I know this was already here, but this is a bit colloquial… It could just be reformatted as a note maybe?